code/game/machinery/computer/arcade.dm

```diff
@@ -80,20 +81,36 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		new /obj/item/gun/energy/pulse/prize(src)
 		visible_message("<span class='notice'>[src] dispenses.. woah, a gun! Way past cool.</span>", "<span class='notice'>You hear a chime and a shot.</span>")
 		SSmedals.UnlockMedal(MEDAL_PULSE, user.client)
+    
+	else
+  
+		new prizeselect(src)
+
+	var/atom/movable/the_prize = pick(contents)
+		visible_message("<span class='notice'>[src] dispenses [the_prize]!</span>", "<span class='notice'>You hear a chime and a clunk.</span>")
 		return
 
+	the_prize.forceMove(get_turf(src))
+
```

Comment:
this is error because the indentation is fucked. Also you have a return before the last line of the proc.

Reply:
ah